assets/prototype/domain/eventEditor/containers/queries/useDateItem.js

```diff
@@ -3,15 +3,20 @@ import { GET_DATETIME } from './dates';
 
 const useDateItem = ({ id }) => {
 	const client = useApolloClient();
+	let data;
 
-	const data = client.readQuery({
-		query: GET_DATETIME,
-		variables: {
-			id,
-		},
-	});
+	try {
+		data = client.readQuery({
+			query: GET_DATETIME,
+			variables: {
+				id,
+			},
+		});
+	} catch (error) {
+		data = {};
+	}
```

Comment:
I assume this is a temp change @manzoorwanijk ?

Reply:
Yes, a temporary fix.